app/controllers/home_controller.rb

```diff
@@ -5,6 +5,8 @@ class HomeController < ApplicationController
   skip_verify_authorized only: :show
 
   def show
+    redirect_to dashboard_path if allowed_to?(:show?, :dashboard)
```

Comment:
Here's a problem: This means the user can't get to the homepage, e.g., by clicking on "Stanford Digital Repository" in the header.

@amyehodge Is it acceptable that the user can't get to the home page?

Reply:
This makes me wonder whether https://github.com/sul-dlss/hungry-hungry-hippo/issues/830 came out of user testing or if it's something we thought useful. Maybe it's more confusing than useful?

Reply:
It came from me being annoyed that I had to click "Enter here" every time I went to the site.

Reply:
@mjgiarlo @justinlittman I hadn't realized this would prevent them from ever getting there via clicking on the link in the header. I hadn't thought that through. I do think that's not desirable. Maybe we should discuss.

Reply:
@justinlittman thank you for adding hold, I meant to for this reason, thinking it needed discussion.

Reply:
Maybe we should set this aside until one of our production or testing users asks for it.

Reply:
@mjgiarlo I'm totally OK with that. I'm going to draft this just so there is no expectation of it being ready to merge.